directives/point_of_interest.py

```diff
@@ -54,6 +55,7 @@ class PointOfInterest(Directive):
         # not_in_slides and not_in_book are used with the presentation maker
         'not_in_slides': directives.flag,
         'not_in_book': directives.flag,
+        'no_poi_box': directives.flag,
```

Comment:
I would use hyphens `-` in option names instead of underscores `_`, but the existing `not_in_slides` option has already been named like that. I guess we keep underscores here.

Reply:
Okay.